Tests/SonarScanner.MSBuild.PreProcessor.Test/RoslynAnalyzerProviderTests.cs

```diff
@@ -93,7 +93,7 @@ public void RoslynConfig_NoAssemblies()
         // Arrange
```

Comment:
It's fine to leave one empty line after ALL scaffolding and act, before all assertions. Every other should go away.

Reply:
Lines 312-369 are particularly bad. Are those assertions afraid of each other or what?

Reply:
This comment was in `RoslynConfig_NoAssemblies` that also needs a bit of shaving